lighthouse-core/lib/timing-trace-saver.js

```diff
@@ -5,70 +5,48 @@
  */
 'use strict';
 
-/**
- * Technically, it's fine for usertiming measures to overlap, however non-async events make
- * for a much clearer UI in traceviewer. We do this check to make sure we aren't passing off
- * async-like measures as non-async.
- * prevEntries are expected to be sorted by startTime
- * @param {LH.Artifacts.MeasureEntry} entry user timing entry
- * @param {LH.Artifacts.MeasureEntry[]} prevEntries user timing entries
- */
-function checkEventOverlap(entry, prevEntries) {
-  for (const prevEntry of prevEntries) {
-    const prevEnd = prevEntry.startTime + prevEntry.duration;
-    const thisEnd = entry.startTime + entry.duration;
-    const isOverlap = prevEnd > entry.startTime && prevEnd < thisEnd;
-    if (isOverlap) {
-      // eslint-disable-next-line no-console
-      console.error(`Two measures overlap! ${prevEntry.name} & ${entry.name}`);
-    }
-  }
-}
-
 /**
  * Generates a chromium trace file from user timing measures
  * Adapted from https://github.com/tdresser/performance-observer-tracing
  * @param {LH.Artifacts.MeasureEntry[]} entries user timing entries
- * @param {string=} trackName
+ * @param {number=} threadId
  */
-function generateTraceEvents(entries, trackName = 'measures') {
+function generateTraceEvents(entries, threadId = 0) {
   if (!Array.isArray(entries)) return [];
 
   /** @type {LH.TraceEvent[]} */
   const currentTrace = [];
-  let id = 0;
-
   entries.sort((a, b) => a.startTime - b.startTime);
   entries.forEach((entry, i) => {
-    checkEventOverlap(entry, entries.slice(0, i));
-
     /** @type {LH.TraceEvent} */
-    const traceEvent = {
-      name: entry.name,
-      cat: entry.entryType,
+    const startEvt = {
+      // 1) Remove 'lh:' for readability
+      // 2) Colons in user_timing names get special handling in traceviewer we don't want. https://goo.gl/m23Vz7
+      //    Replace with a 'Modifier letter colon' ;)
+      name: entry.name.replace('lh:', '').replace(/:/g, '꞉'),
```

Comment:
😆 

maybe do an explicit `.replace(/:/g, '\ua789'),` just to make it doubly clear that the character is different?